lib/menu.rb

```diff
@@ -0,0 +1,60 @@
+
+class Menu
+
+  attr_reader :basket, :receipt_list, :checkout
+
+  MEALS_LIST =
```

Comment:
I think an array of hashes might have been a more suitable data structure, because instead of hardcoding menu numbers as keys you can take advantage of the array object's built in indexing